### PR TITLE
Adding outboundvoiceprofiles functionality and examples.

### PR DESCRIPTION
**Pull request by rajjy1**

### CODE REVIEW

File: src/Telnyx.net/Services/OutboundVoiceProfiles/OutboundVoiceProfileService.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        IUpdatable<OutboundVoiceProfile, CreateOutboundVoiceProfileOptions>
+    {
+        public override string BasePath => "/outbound_voice_profiles";
+
+        public OutboundVoiceProfile Create(CreateOutboundVoiceProfileOptions createOptions, RequestOptions requestOptions = null)
+        {
+            return this.CreateEntity(createOptions, requestOptions);
+        }
+
+        public Task<OutboundVoiceProfile> CreateAsync(CreateOutboundVoiceProfileOptions createOptions, RequestOptions requestOptions = null, CancellationToken cancellationToken = default)
```
Comment (thedonmon) [inline src/Telnyx.net/Services/OutboundVoiceProfiles/OutboundVoiceProfileService.cs:22]: `public async Task<`

Comment (rajjy1) [reply src/Telnyx.net/Services/OutboundVoiceProfiles/OutboundVoiceProfileService.cs:22]: Changed.

**Comment by rajjy1** [comment]
> Are there any tests for this?

Added unit tests

File: src/TelnyxTests/Services/OutboundVoiceProfiles/OutboundVoiceProfileServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+            this.cancellationToken = default(CancellationToken);
+        }
+
+        [Fact]
+        public void Create()
+        {
+            var result = this.service.Create(this.createOptions);
+            Assert.NotNull(result);
+            Assert.Equal("Telnyx.OutboundVoiceProfile", result.GetType().ToString());
```
Comment (thedonmon) [inline src/TelnyxTests/Services/OutboundVoiceProfiles/OutboundVoiceProfileServiceTest.cs:52]: Tests are failing. Change these lines to 
`Assert.Equal(typeof(OutboundVoiceProfile), result.GetType());`

Comment (rajjy1) [reply src/TelnyxTests/Services/OutboundVoiceProfiles/OutboundVoiceProfileServiceTest.cs]: Changed.

**Pull request merged**